pixels/endpoints/general.py

```diff
@@ -0,0 +1,175 @@
+import logging
+
+from fastapi import APIRouter, Depends, HTTPException, Request, Response
+
+from pixels.constants import Ratelimits, Sizes
+from pixels.models import GetSize, Message, Pixel
+from pixels.utils import auth, ratelimits
+
+log = logging.getLogger(__name__)
+router = APIRouter(tags=["Canvas Endpoints"], dependencies=[Depends(auth.JWTBearer())])
+
+
+@router.get("/size", response_model=GetSize)
+async def size(request: Request) -> GetSize:
+    """
+    Get the size of the pixels canvas.
+
+    You can use the data this endpoint returns to build some cool scripts
+    that can start the ducky uprising on the canvas!
+
+    This endpoint doesn't require any authentication so dont worry
+    about giving any headers.
+
+    #### Example Python Script
+    ```py
+    import requests
+
+    r = requests.get("https://pixels.pythondiscord.com/get_size")
```

Comment:
Forgot to fix the example here
```suggestion
    r = requests.get("https://pixels.pythondiscord.com/size")
```